client/src/components/Friend.jsx

```diff
@@ -1,31 +1,29 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
+import regeneratorRuntime from 'regenerator-runtime';
```

Comment:
Good to know!  Does async/await throw errors without this or just not work?  I think I had some issues trying to use async but ended just bailing for nested then statments

Reply:
Yeah, I got errors. Researched and led me to this. The package is from FB, so seemed like a low-risk add.